2.1-beer-layout/style.css

```diff
@@ -1,7 +1,143 @@
 /* apply a natural box layout model to all elements, but allowing components to change */
 html {
   box-sizing: border-box;
+  border: .2em solid #38A3D7;
 }
 *, *:before, *:after {
   box-sizing: inherit;
 }
+
+div {
+	text-align: center;
+	font-family: 'Lato', sans-serif;
+}
+.beer {
+	width:33%;
+	display: inline-block;
```

Comment:
When using `display: inline-block` look at this article: https://css-tricks.com/what-is-vertical-align/.
Right now your grid items are different heights and that is why you have the jumping up and down.